### ISSUE TITLE
Alt Text Quality accessibility rule not enabled by default

### ISSUE THREAD

**Issue by albinazs** [wagtail/wagtail#12636]
### Issue Summary

In Wagtail 6.2 and above, the new [alt text quality](https://docs.wagtail.org/en/stable/releases/6.2.html#alt-text-accessibility-check) accessibility rule is not enabled by default, contrary to what was indicated in the [documentation](https://docs.wagtail.org/en/stable/advanced_topics/accessibility_considerations.html#built-in-accessibility-checker).
This issue affects all projects that do not have a custom `AccessibilityItem`, similar to what is implemented in the [Bakery demo](https://github.com/wagtail/bakerydemo/blob/main/bakerydemo/base/wagtail_hooks.py#L35-L44).

A potential solution could be to include this rule in the list of [default Axe rules](https://github.com/wagtail/wagtail/blob/main/wagtail/admin/userbar.py#L49), enabling it by default for all projects. Alternatively, each project would need to incorporate the rule manually.

### Steps to Reproduce

1. Open any Wagtail 6.2 or later project that doesn't have a custom `AccessibilityItem` (**not** a Bakery demo)
2. Add an image with alt text that is considered anti-pattern (`<img alt="Image.jpg" ...  </img>`)
3. Check the output of the accessibility checker - the image won't be flagged

-   I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: (yes)

### Technical details

-   Wagtail version: 6.2+

### Working on this

Anyone can contribute to this. View our [contributing guidelines](https://docs.wagtail.org/en/latest/contributing/index.html), add a comment to the issue once you’re ready to start.

**Comment by thibaudcolas** [comment]
Thank you @albinazs, confirmed! I can’t recall our decision-making here, whether it was intentional for us to ship it as an opt-in feature or no. In any case it probably makes sense for it to be enabled by default now that alt text support has improved greatly in v6.3, and with more refinements in v6.4. #12793
